source/disassemble.cpp

```diff
@@ -265,7 +265,11 @@ void Disassembler::EmitOperand(const spv_parsed_instruction_t& inst,
     case SPV_OPERAND_TYPE_BUILT_IN:
     case SPV_OPERAND_TYPE_GROUP_OPERATION:
     case SPV_OPERAND_TYPE_KERNEL_ENQ_FLAGS:
-    case SPV_OPERAND_TYPE_KERNEL_PROFILING_INFO: {
+    case SPV_OPERAND_TYPE_KERNEL_PROFILING_INFO:
+    case SPV_OPERAND_TYPE_DEBUG_BASE_TYPE_ATTRIBUTE_ENCODING:
+    case SPV_OPERAND_TYPE_DEBUG_COMPOSITE_TYPE:
+    case SPV_OPERAND_TYPE_DEBUG_TYPE_QUALIFIER:
+    case SPV_OPERAND_TYPE_DEBUG_OPERATION: {
```

Comment:
Maybe the whole list can be checked using `spvOperandIsConcrete()`?

Reply:
Not here.  This stack of cases is for things that are enums that are not bit-flags.

There is a potential cleanup here and other places to sequences of if/else where each branch is a check via a function like spvOperandIsValueEnum and spvOperandIsBitEnum and so on.    (Or to make yet another more abstract set of enums.)   But I don't want to make that change as part of this CL.